.github/workflows/pre-commit.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       max-parallel: 4
       matrix:
         python-version: [2.7]
```

Comment:
should this be converted to 3.8?

Reply:
We run in both 2.7 and 3.8, 3.8 is specified elsewhere in the file